build-config/scripts/make-devices.pl

```diff
@@ -168,18 +168,11 @@ sub devices
     mkdir "${devdir}pts", 0755;
 
     print "NOR flash access: mtd mtdblock\n";
-    foreach my $i (0..7) {
+    foreach my $i (0..12) {
 	makedev("mtdblock${i}", "b", 31, ${i},     ${root}, ${root}, 0660);
 	makedev("mtd${i}",      "c", 90, (2*${i}), ${root}, ${root}, 0660);
     }
 
-    print "UBI device access: ubi\n";
-    makedev("ubi_ctrl", "c", 10, 63, ${root}, ${root}, 0660);
-    makedev("ubi0"    , "c", 253, 0, ${root}, ${root}, 0660);
-    foreach my $i (0..7) {
-	makedev("ubi0_${i}", "c", 253, ${i}+1, ${root}, ${root}, 0660);
-    }
-
```

Comment:
Yeah - I was wondering about that. I figured that legacy PPC platforms will also be broken but that thought may be flawed.
I'll remove (or reinsert) that part and look into a more flexible way of doing it generally. I followed those links above but I'm not sure they will work at build time - I think you need to create the nodes once the switch has booted. Been a while since I looked at the ONIE boot scripts.....time to go refresh my memory